stanford/stage/analytics-jenkins1.yml

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ansible-playbook
+---
+- include: ../playbooks/analytics-jenkins.yml
+  vars:
+    CLUSTER_INSTANCE_TYPE: t2.medium
```

Comment:
Since this is stage, we can drop this line and fall back to the default, `.t2.micro`, I think?